.rubocop.yml

```diff
@@ -1,3 +1,12 @@
 inherit_gem:
   citizens-advice-style:
     - default.yml
+
+inherit_mode:
```

Comment:
This in "theory" shouldn't be needed. As primarily it's used for when we have multiple sources. But I "might" be wrong with this.